modules.d/01systemd-tmpfiles/module-setup.sh

```diff
@@ -60,7 +63,9 @@ install() {
             "$systemdsystemconfdir"/systemd-tmpfiles-setup.service \
             "$systemdsystemconfdir/systemd-tmpfiles-setup.service.d/*.conf" \
             "$systemdsystemconfdir"/systemd-tmpfiles-setup-dev.service \
-            "$systemdsystemconfdir/systemd-tmpfiles-setup-dev.service.d/*.conf"
+            "$systemdsystemconfdir/systemd-tmpfiles-setup-dev.service.d/*.conf" \
+            "$systemdsystemconfdir"/systemd-tmpfiles-setup-dev-early.service \
```

Comment:
Are you sure that the service file also needs to be present below "$systemdsystemconfdir" e.g.:  /etc/systemd/system?
Normally, that is only the case when the service file deviates from the shipped one, e.g. when it is customized.
Otherwise, installing: "$systemdsystemunitdir"/systemd-tmpfiles-setup-dev-early.service  should be enough.

However, I see this for other service files too.
So, maybe there is a good reason, any idea?

Reply:
Yes, they are not strictly necessary, I just cherry-picked the upstream commit. I think all these paths under `$systemdsystemconfdir` were added in the new `systemd-` submodules just for completeness.

Note that these customized paths are only added with the `systemd-tmpfiles` dracut module, which is only included to the initrd if it's explicitly added (there is no other module adding it as a dependency at this time). So, In a normal situation, only the service under `$systemdsystemunitdir` is installed (only the `systemd` dracut module is included).

Reply:
Ok, thanks for the background, then I'm fine with it.
I don't expect many users having customized unit files in the initrd anyway.